packages/models-library/src/models_library/api_schemas_directorv2/comp_runs.py

```diff
@@ -62,20 +63,16 @@ class ComputationRunRpcGetPage(NamedTuple):
     total: PositiveInt
 
 
-def _none_to_zero_float_pre_validator(value: Any):
-    if value is None:
-        return 0.0
-    return value
-
-
 class ComputationTaskRpcGet(BaseModel):
     project_uuid: ProjectID
     node_id: NodeID
     state: RunningState
-    progress: Annotated[float, BeforeValidator(_none_to_zero_float_pre_validator)]
+    progress: float
```

Comment:
this disappears? not sure whether you need this? any test reflecting that? or why was it added in the first place?